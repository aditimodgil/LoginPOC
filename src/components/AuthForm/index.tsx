import React from "react";
import { StyledFormWrapper, StyledForm } from "../helper";
import { IAuthFormProps } from "../../types";

const AuthForm = ({ submit , children }: IAuthFormProps) => (
  <StyledFormWrapper>
    <StyledForm onSubmit={submit}>{children}</StyledForm>
  </StyledFormWrapper>
);

export default AuthForm;
