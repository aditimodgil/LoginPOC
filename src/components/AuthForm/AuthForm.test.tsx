import React from 'react';
import AuthForm from './index';
import {shallow} from 'enzyme';


describe('<AuthForm />', ()=>{
    const props ={
      submit : jest.fn(),
      children : React.Children
    }
    it('should render without errors', ()=>{
      const wrapper = shallow(<AuthForm {...props} />);
      expect(wrapper.find('AuthForm')).toMatchSnapshot();

    });
})
