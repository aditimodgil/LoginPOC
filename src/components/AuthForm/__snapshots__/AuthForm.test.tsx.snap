// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AuthForm /> should render without errors 1`] = `ShallowWrapper {}`;
